### ISSUE TITLE
[PyTorch/NotImplementedError]: The following operators are not implemented: ['aten::bitwise_and']

### ISSUE THREAD

**Issue by HanGuo97** [apache/tvm#16005]
### Steps to reproduce

```python
import torch
def func(x1, x2):
    return torch.bitwise_and(x1, x2)

x1 = torch.tensor([True, True, False], device="cuda")
x2 = torch.tensor([False, True, False], device="cuda")
func(x1, x2) # --> runs
torch.compile(func, backend="tvm")(x1, x2) # --> not implemented error
```

### Triage

Please refer to the list of label tags [here](https://github.com/apache/tvm/wiki/Issue-Triage-Labels) to find the relevant tags and add them below in a bullet format (example below).

* needs-triage

**Comment by mshr-h** [comment]
I'll work on it.
